docs/models/components/extra.md

```
@@ -0,0 +1,21 @@
+let value: Extra = {
+  name: "USD Coin",
+  version: "2",
+};
+```
+
+## Fields
+
+| Field              | Type               | Required           | Description        | Example            |
+| ------------------ | ------------------ | ------------------ | ------------------ | ------------------ |
+| `name`             | *string*           | :heavy_minus_sign: | N/A                | USD Coin           |
+| `version`          | *string*           | :heavy_minus_sign: | N/A                | 2                  |
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)